.travis.yml

```diff
@@ -6,17 +6,18 @@ node_js:
 cache:
   npm: false
 before_install:
-- '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ ! -z ${encrypted_ac3aacad7ba8_key} ] &&
-  openssl aes-256-cbc -K $encrypted_ac3aacad7ba8_key
-  -iv $encrypted_ac3aacad7ba8_iv -in secrets.tar.enc -out test/resources/secrets.tar
-  -d || true'
-- '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ ! -z ${encrypted_ac3aacad7ba8_key} ]
-  && cd test/resources/ && tar xvf secrets.tar
-  && cd ../.. || true'
+- openssl aes-256-cbc -K $encrypted_02d9f3c197ce_key -iv $encrypted_02d9f3c197ce_iv
```

Comment:
You may want to confirm this because I could be wrong - but by removing the "if" statement about it not being a pull request, it looks like the `auth.js` file will be created and that would mean the integration tests would be run. We do not want to run the integration tests on PR builds, only Push builds.

Whenever I updated the credentials, I would just drop the changes to the `.travis.yml` file - they are just formatting changes and are not needed. You can probably do the same here. The only thing I'm not sure about is this new key (`$encrypted_02d9f3c197ce_key`) - I'm not sure why that's there or how you need to included it